LocationManagerExample/LocationManagerExample/INTUViewController.m

```diff
@@ -76,23 +92,12 @@ - (void)startLocationUpdateSubscription
         if (status == INTULocationStatusSuccess) {
             // A new updated location is available in currentLocation, and achievedAccuracy indicates how accurate this particular location is
             strongSelf.statusLabel.text = [NSString stringWithFormat:@"Subscription block called with Current Location:\n%@", currentLocation];
+            return;
         }
-        else {
-            // An error occurred, which causes the subscription to cancel automatically (this block will not execute again unless it is used to start a new subscription).
-            strongSelf.locationRequestID = NSNotFound;
-            
-            if (status == INTULocationStatusServicesNotDetermined) {
-                strongSelf.statusLabel.text = @"Error: User has not responded to the permissions alert.";
-            } else if (status == INTULocationStatusServicesDenied) {
-                strongSelf.statusLabel.text = @"Error: User has denied this app permissions to access device location.";
-            } else if (status == INTULocationStatusServicesRestricted) {
-                strongSelf.statusLabel.text = @"Error: User is restricted from using location services by a usage policy.";
-            } else if (status == INTULocationStatusServicesDisabled) {
-                strongSelf.statusLabel.text = @"Error: Location services are turned off for all apps on this device.";
-            } else {
-                strongSelf.statusLabel.text = @"An unknown error occurred.\n(Are you using iOS Simulator with location set to 'None'?)";
-            }
-        }
+        // An error occurred, which causes the subscription to cancel automatically (this block will not execute again unless it is used to start a new subscription).
+        strongSelf.locationRequestID = NSNotFound;
+        NSString *statusDescription = [strongSelf getStatusDescription:status];
+        strongSelf.statusLabel.text = statusDescription;
```

Comment:
No need for the local `statusDescription` variable. Just:
    strongSelf.statusLabel.text = [strongSelf getStatusDescription:status];